### ISSUE TITLE
Ideas for the Town Environment?

### ISSUE THREAD

**Issue by quigley-c** [codeandkey/boom#118]
# What is the purpose of this issue?
This issue is for throwing around and and all ideas related to the Town area of the game. Anything in this thread is open for discussion and participation is encouraged.

## What kinds of things should be discussed here?
This issue should be used to propose ideas for level design, backgrounds, scenery, music, story events.. anything related directly to the town area and how the player experiences it.

**Comment by codeandkey** [comment]
I agree with everything here so far. I've enjoyed hub areas with calming music as they often act as a checkpoint between major tasks in the game.
I'm a big fan of the hub music present in [HLD](https://youtu.be/_iZw-PejWks?t=528) and [Amnesia](https://www.youtube.com/watch?v=tWhUPlFnAe8), however those games both have darker themes.
